README.md

```diff
@@ -50,7 +50,8 @@ Please send us a pull request with your suggestions.
 
 #### Doesn't this impact the right to free speech? Why are you stomping on my rights?
 
-See this XKCD for our feelings on this matter: https://xkcd.com/1357/
+In the many countries that do have the right to free speech, this means (with obvious exeptions) the government cannot arrest you for saying anything.  Many countries contain free speech in their constitution which can only apply to the government for enforcing or creating laws.
+As most companies and organizations are owned by private individuals, they are only restricted by laws that apply to people and their businesses.  The free speech sections contained in constitutions only apply to the government, restricting the kinds of laws that it can make.  So no, this does not impact your right to free speech.
```

Comment:
@bkeepers I did not realize that, thank you for the clarification.

@Addvilz I agree.  Pretty much any move Github makes these days could have a huge effect on software.  They could dramatically align every open source project into the direction of one open source software license if they just encouraged automatic adding of a license to a repo...

 [Off-topic idea:  Being able to pick and add an open source license or Code of Conduct to your project from a list of options would be a pretty cool feature!]